modin/pandas/groupby.py

```diff
@@ -683,6 +685,12 @@ def groupby_on_multiple_columns(df):
         return self._df._default_to_pandas(groupby_on_multiple_columns)
 
 
+if os.environ.get("MODIN_EXPERIMENTAL", "").title() == "True":
```

Comment:
Always for experimental? What if user wants to use other experimental capabilities?

Reply:
Also is this needed for `SeriesGroupBy` as well?

Reply:
If a user does not activate a remote context this doesn't change much.
Sure it adds a little of overhead (a few extra calls) in instantiation, and a tiny overhead in class-level attribute access (because those now go via metaclass `__getattribute__`), but this is unavoidable for now - there's no way to replace a metaclass of a class at runtime, and we don't know if a user would be activating remote context or not during importing, where dual-nature classes must be set up.